### DISCUSSION TITLE
Newbie Question: Controlling order of tile rendering when tile size is bigger than grid size

### DISCUSSION THREAD

**Discussion by digitalbricklayer** [StarArawn/bevy_ecs_tilemap#459] [Q&A]
I have a tilemap where the `TilemapTileSize` is larger than `TilemapGridSize`. This is due to the terrain on the tiles sometimes overflows the top of the tile like when a mountain is part of the terrain.

The top row of the tilemap is displayed properly but all rows underneath clip the top of the mountain off with the bottom of the tile underneath.

I'm assuming this is pretty normal given that the plugin allows me to have a tile size larger than the grid size. I need to be able to control the order of rendering. If rendering happened on the top rows first and then worked down, this problem would be avoided.

I assumed that render order would be the same as the order in which the `TileBundle` are added to the ECS. But that can't be the case because I am working from the first row downwards when adding the tiles ie from `TilePos { 0, 0 }` through `TilePos { 0, 31 }` and so on down the rows to `TilePos { 31, 31 }`.

Any help much appreciated.

**Comment by doonv** [comment]
I'm experiencing a similar issue. No solution to it though.
